src/game/Tactical/Turn_Based_Input.cc

```diff
@@ -2413,12 +2395,6 @@ void GetKeyboardInput(UIEventKind* const puiNewEvent)
 			}
 			else if ( InputEvent.usKeyState & CTRL_DOWN )
       {
```

Comment:
I'm open to putting this as a bug on the backlog, and merging what's done into the mainline.  That way we don't lose the task, and we can shut the branch down quickly.